### PR TITLE
Use common staged manager

### PR DESCRIPTION
**Pull request by piotrmiskiewicz**
**Description**

Changes proposed in this pull request:
- use staged manager in the deprovisioning process

**Related issue(s)**

### CODE REVIEW

**Comment by piotrmiskiewicz** [comment]
should not br merged before 21th september

File: components/kyma-environment-broker/internal/process/deprovisioning/remove_runtime_test.go [hunk 1]
```diff
@@ -32,50 +33,16 @@ func TestRemoveRuntimeStep_Run(t *testing.T) {
 
 		// when
 		entry := log.WithFields(logrus.Fields{"step": "TEST"})
-		result, repeat, err := step.Run(operation, entry)
+		result, repeat, err := step.Run(operation.Operation, entry)
 
 		// then
 		assert.NoError(t, err)
-		assert.Equal(t, 1*time.Second, repeat)
+		assert.Equal(t, 0*time.Second, repeat)
```
Comment (piotrmiskiewicz) [inline components/kyma-environment-broker/internal/process/deprovisioning/remove_runtime_test.go:40]: the old implementation of that step returns 1 second if everything was ok, then the check if provisioner finished was done in the initialisation step. This PR makes it simpler

File: components/kyma-environment-broker/internal/process/operation_manager.go [hunk 2]
```diff
@@ -99,6 +99,9 @@ func (om *OperationManager) UpdateOperation(operation internal.Operation, update
 		log.Errorf("while updating operation: %v", err)
 		return operation, 1 * time.Minute, err
 	}
+	if op == nil {
+		fmt.Println("XXXXXX")
```
Comment (jaroslaw-pieszka) [inline components/kyma-environment-broker/internal/process/operation_manager.go:103]: Should not we log something properly?

**Review by jaroslaw-pieszka — APPROVED** [review]

**Review by jaroslaw-pieszka — APPROVED** [review]

**Pull request merged**